mkdocs/themes/mkdocs/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -1,92 +1,100 @@
-# French translations for mkdocs/themes/mkdocs.
-# Copyright (C) 2019 mkdocs
-# This file is distributed under the same license as the mkdocs/mkdocs project.
-# Antoine Lauzon, 2019.
+# French translations for PROJECT.
+# Copyright (C) 2021 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
```

Comment:
What was all the meta data stripped out and replaced with placeholders?

I'm guessing this is because the `en` file was deleted which this file was based on. However, we still need a template, which should have a `.pot` extension. The template should include all of the meta data for the MkDocs project so that when specific language files are generated, that data is copied over. And, of course, the various commands defined in the makefile would need to point to that template.

Reply:
I dropped it using the `--omit-header` since I did not see any point having it in the project at first glance. But the pybabel commands are not consistent so indeed it does not propagate back to the initialized .po files which lead to this inconsistency.

I'll fix that as I try to move to setup.py to address your idea below.